modules/apps/auxano-theme/src/templates/portal_normal.ftl

```diff
@@ -0,0 +1,188 @@
+<!DOCTYPE html>
+
+<#include init />
+
+<html class="${root_css_class}" dir="<@liferay.language key="lang.dir" />" lang="${w3c_language_id}">
+
+<head>
+	<title>${the_title} - ${company_name}</title>
+
+	<meta content="initial-scale=1.0, width=device-width" name="viewport" />
+
+	<@liferay_util["include"] page=top_head_include />
+</head>
+
+<body class="${css_class}">
+
+<@liferay_ui["quick-access"] contentId="#main-content" />
+
+<@liferay_util["include"] page=body_top_include />
+
+<@liferay.control_menu />
+
+
+<div class="container-fluid" id="wrapper">
+	<header id="banner" role="banner">
+
+
+
+		<!-- Landing page description for company -->
+		<h2 class="landing-description">growing and increasing as a <strong>family</strong> <br> one cup at a time.</h2>
+		<div id="heading">
+			<h1 class="site-title">
+			
+				<#if show_site_name>
+					<span class="site-name" title="<@liferay.language_format arguments="${site_name}" key="go-to-x" />">
+						Auxano
```

Comment:
Site name should be a Freemarker variable rather than hardcoded.  You should be a be able to use ```${site_name}``` for this.